app/routes/content/index.js

```diff
@@ -107,33 +110,52 @@ export default Route.extend(Analytics, ResetScrollMixin, SetupSubmitControllerMi
         if (themeId === providerId) {
             return Promise.all([
                 provider,
-                preprint.get('node'),
```

Comment:
Not loading this here, because if the supplemental project was private, a 403 would cause the entire page to 404.  We don't care if the supplemental project is private; it shouldn't affect the preprint itself from rendering.